### PR TITLE
NAS-112741 / None / Removed welcome dialog and added "Help" widget on dashboard

### PR DESCRIPTION
**Pull request by RehanY147**
To test
- Make sure that the help widget shows up by default on the dashboard page
- Make sure that it's configurable and hides when unchecked on the Dashboard Configure form
- Check responsive on phone screens

### CODE REVIEW

File: src/app/pages/dashboard/components/widget-help/widget-help.component.ts [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    });
+  }
+
+  ngOnInit(): void {
+    this.sysGenService.getProductType$.pipe(untilDestroyed(this)).subscribe((productType) => {
+      this.systemType = productType as ProductType;
+    });
+  }
+
+  turnOffWelcomeDialog(): void {
```
Comment (AlexKarpov98) [inline src/app/pages/dashboard/components/widget-help/widget-help.component.ts:44]: Good work!
Image in ticket
image

Web UI
Screenshot_7

Screenshot_8

There is some difference in UI. Not sure to add a logo like the Welcome dialog.
Missing unit testing.
Please remove all prev codes of the Welcome dialog.

**Review by AlexKarpov98 — CHANGES_REQUESTED** [review]
Good work!
Image in ticket
image

Web UI
Screenshot_7

Screenshot_8

There is some difference in UI. Not sure to add a logo like the Welcome dialog.
Missing unit testing.
Please remove all prev codes of the Welcome dialog.

**Comment by RehanY147** [comment]
@AlexKarpov98 All of the content on this widget is static except for the product type. Nothing to test. No actions can be performed except link clicks. Also, the ticket suggests that instead of having a form there to subscribe to the newsletter we can just redirect the user to the page on the website for subscription.
Extra code removed. Thanks for pointing that out!

**Review by undsoft — APPROVED** [review]
Looks good to me too. Maybe in the future we should consider having a faster way to remove the widget from dashboard.

**Comment by undsoft** [comment]
backport

**Pull request merged**